www/js/goals.js

```diff
@@ -864,6 +871,81 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
         });
     }
 
+    var showUserId = function() {
+        console.log("Showing user id");
+        $ionicPopup.show({
+          title: 'Bic2Cal Survey',
+          templateUrl: 'templates/goals/uid.html',
+          scope: $scope,
+            buttons: [{
+              text: 'Copy user id and open survey',
+              type: 'button-positive',
+              onTap: function(e) {
+                $cordovaClipboard.copy($scope.profile.id).then(function () {
```

Comment:
I tried all that, and nothing gets copied to the clipboard. But now I tried again (without changing absolutely nothing), and it works! haha